server/schema/query.js

```diff
@@ -0,0 +1,23 @@
+const bookQuery = `
+    books(page: Int, limit: Int, filter: BookFilter): PaginatedBooks!
+    book(id: ID!): Book
+`;
+
+const authorQuery = `
+    authors(page: Int, limit: Int, filter: AuthorFilter): PaginatedAuthors!
+    author(id: ID!): Author
+`;
+
+const reviewQuery = `
+    reviews(book_id: ID!): [Review]
```

Comment:
why not bookId?

Reply:
Because this is supposed to return the review array (it's fetching all the reviews).